### DISCUSSION TITLE
beef + ngrok

### DISCUSSION THREAD

**Discussion by wandelguina** [frontendbr/forum#1054] [Perguntas e Respostas]
alguém pode me ajudar.estou tentando usar o beef + ngrok em rede externa.mais quando abro o link do ngrok da vitima ele nn sobe no painel.ja tentei em rede interna sem o ngrok e funfo.mais quero em rede externa .desde ja agradeço

**Comment by gerardo-junior** [comment]
pode ser alguma configuração que ta errada, porque deveria funcionar perfeitamente.

eu ultimamente deixei de usar ngrok para usar o [serveo](http://serveo.net/) acho mais limpo.

mas pra um atack assim recomendo uma cloud ou um ddns a latencia vai ser melhor se bem configurado.
